index.js

```diff
@@ -1,9 +1,35 @@
 'use strict';
 
-var murmurhash = require('murmurhash');
+function get(arr, key) {
+	return first(arr, key, function (i) {
+		return arr[i].value;
+	});
+}
+
+function set(arr, key, value) {
+	var found = first(arr, key, function (i) {
```

Comment:
not really. first here finds the first occurrence of the pair. get only gets the value.